themes/theme-gmd/index.jsx

```diff
@@ -11,12 +11,19 @@ import React from 'react';
 import { render } from 'react-dom';
 import { configureStore } from '@shopgate/pwa-common/store';
 import { appWillStart } from '@shopgate/pwa-common/action-creators/app';
+import { i18n } from '@shopgate/engage/core';
 import smoothscroll from 'smoothscroll-polyfill';
 import fetchClientInformation from '@shopgate/pwa-common/actions/client/fetchClientInformation';
+import locales from './locale';
 import reducers from './pages/reducers';
 import subscribers from './pages/subscribers';
 import Pages from './pages';
 
+i18n.init({
```

Comment:
Since you already initialized `i18n`, could you change I18nProvider.getI18nInstance to use already curried functions? Otherwise we will have 2 i18n systems with caches in App.

What about themes/theme-gmd/pages/index.jsx:56 ? Do we still pass locales throught App ? I guess from now on, it is reduntant and can be removed

Reply:
If it's implemented in the common helpers then it would be obsolete.